apps/gitness/src/pages/repo/repo-list.tsx

```diff
@@ -37,17 +38,16 @@ export default function ReposListPage() {
       data && (
         <RepoList
           LinkComponent={LinkComponent}
-          // @ts-expect-error remove "@ts-expect-error" once type issue for "content" is resolved
-          repos={data?.content?.map(repo => {
+          repos={data?.map((repo: RepoRepositoryOutput) => {
```

Comment:
`repo: RepoRepositoryOutput`

this typecast should not be needed.. is the `data` not already an array of the correct type?

Reply:
This is actually a weird scenario. The API returns a response of type `TypesRepository[]` by definition. But the actual response returned by the api endpoint is of type `RepoRepositoryOutput`. I observed it when I was looking for `is_public` attribute, which isn't present on type `TypesRepository`.

Reply:
I will start a thread for this.